Source/JavaScriptCore/runtime/StringPrototype.cpp

```diff
@@ -1028,9 +1028,10 @@ static inline char32_t codePointAt(const String& string, unsigned position, unsi
 {
     RELEASE_ASSERT(position < length);
     if (string.is8Bit())
-        return string.characters8()[position];
+        return string.span8()[position];
     char32_t character;
-    U16_NEXT(string.characters16(), position, length, character);
+    auto characters = string.span16();
```

Comment:
Why the local variable?

Reply:
Didn't we establish that the local variable was needed or it fails building on Mac-ews? (though it builds fine locally).
__u2 unused variable. You said last time that it was fixed but it still doesn't work on EWS for some reason.